homeassistant/components/sensor/template.py

```diff
@@ -30,7 +30,7 @@
     vol.Optional(CONF_ENTITY_PICTURE_TEMPLATE): cv.template,
     vol.Optional(CONF_FRIENDLY_NAME_TEMPLATE): cv.template,
     vol.Optional(ATTR_FRIENDLY_NAME): cv.string,
-    vol.Optional(ATTR_UNIT_OF_MEASUREMENT): cv.string,
+    vol.Optional(CONF_UNIT_OF_MEASUREMENT_TEMPLATE): cv.template,
```

Comment:
Do not remove `unit_of_measurement` - only add a _new_ config option like wigh friendly name

Reply:
Thanks, I changed this back.